### PR TITLE
Convert GraphicsControls to be .NET MAUI Handlers

### PR DESCRIPTION
**Pull request by jsuarezruiz**
This PR adds changes to:
- Use the latest Microsoft.Maui.Graphics instead System.Graphics.
- Every GraphicControl is now a .NET MAUI Handler.
- The sample is now a .NET MAUI App.

graphicscontrols-netmaui

### CODE REVIEW

File: src/GraphicsControls/Handlers/CheckBox/CheckBoxHandler.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+		public CheckBoxHandler() : base(DrawMapper, PropertyMapper)
+		{
+
+		}
+
+		public static string[] DefaultCheckBoxLayerDrawingOrder =
+			ViewHandler.DefaultLayerDrawingOrder.ToList().InsertAfter(new string[]
+			{
+				"Background",
```
Comment (IDisposable) [inline src/GraphicsControls/Handlers/CheckBox/CheckBoxHandler.cs:30]: Not that this is particularly related to this PR's purpose, but when I see the same string literal in a file multiple times, I worry about accidental typos getting them out of sync (a very subtle bug to find). I wonder if, _since this is an example of how to do things in MAUI_, we should be using `const string` inside this class for these property names?

Maybe we should even expose theme as properties on `ICheckBoxDrawable` and use the `[nameof]ICheckboxDrawable.Background` pattern we see in line 11?

File: src/GraphicsControls/Handlers/DrawMapper.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		public DrawMapper(DrawMapper chained)
+		{
+			Chained = chained;
+		}
+
+		public DrawMapper? Chained { get; set; }
+
+		public Action<ICanvas, RectangleF, TViewDrawable, TVirtualView> this[string key]
+		{
+			set => genericMap[key] = (c, r, h, v) => value?.Invoke(c, r, (TViewDrawable)h, (TVirtualView)v);
```
Comment (IDisposable) [inline src/GraphicsControls/Handlers/DrawMapper.cs:54]: Might it be nice to spell out the arguments with something clearer?

`(canvas, rect, drawable, virtualView) => value?.Invoke(canvas, rect, (TViewDrawable)drawable, (TVirtualView)virtualView);`

File: src/GraphicsControls/Platform/RectangleExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Linq;
+
+namespace Microsoft.Maui.Graphics.Controls
+{
+	public static class RectExtensions
```
Comment (IDisposable) [inline src/GraphicsControls/Platform/RectangleExtensions.cs:6]: Class name doesn't match file name.

File: src/GraphicsControls/Platform/iOS/NativeGraphicsControlView.cs [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+				if (!IsFirstResponder)
+					BecomeFirstResponder();
+
+				var viewPoints = this.GetPointsInView(evt);
+				GraphicsControl?.StartInteraction(viewPoints);
+				_pressedContained = true;
+			}
+			catch (Exception exc)
+			{
+				Debug.WriteLine("An unexpected error occured handling a touch event within the control.", exc);
```
Comment (IDisposable) [inline src/GraphicsControls/Platform/iOS/NativeGraphicsControlView.cs:39]: Is swallowing the exception intentional?

**Review by Redth — APPROVED** [review]

**Comment by jsuarezruiz** [comment]
Updated the sample to validate different visual states (disabled, etc) and customizations:
maui-graphics-sample

File: src/GraphicsControls/Handlers/DrawMapper.cs (hunk 2 above)
Comment (jsuarezruiz) [reply src/GraphicsControls/Handlers/DrawMapper.cs:54]: Changed.

File: src/GraphicsControls/Platform/RectangleExtensions.cs (hunk 3 above)
Comment (jsuarezruiz) [reply src/GraphicsControls/Platform/RectangleExtensions.cs:6]: Updated.

**Pull request merged**